src/main/java/me/gaminglounge/estats/listener/GetDamage.java

```diff
@@ -25,22 +27,32 @@ public class GetDamage implements Listener {
 
 
     @EventHandler
-    public void onDamagedbBy(EntityDamageByEntityEvent event){
+    public void onDamagedBy(EntityDamageByEntityEvent event){
 
-        //damageTest(event.getDamage(), event.getEntity().getType(), event.getDamager().getName(),event.getDamageSource().getDamageType());
-        //damageDisplay(event.getDamage(),event.getDamager());
         EStats.INSTANCE.entityName.add_(event.getEntity());
-        if(event.getDamager() instanceof Player & event.getEntity() instanceof LivingEntity){
+        if(event.getDamager() instanceof Player
+        || event.getDamager() instanceof Projectile
+        & event.getEntity() instanceof LivingEntity){
```

Comment:
Use `&&` instead of `&`
(line +35)

Reply:
Fixed in https://github.com/GamingLounge-me/EStats/pull/6/commits/4b41a489959254b141a15a9baf330469bd785373